service/src/test/java/bio/terra/workspace/service/resource/controlled/ControlledResourceServiceTest.java

```diff
@@ -435,10 +437,15 @@ private ControlledAiNotebookInstanceResource createDefaultPrivateAiNotebookInsta
             null,
             user.getAuthenticatedRequest());
     jobService.waitForJob(createJobId);
-    return jobService
-        .retrieveJobResult(
-            createJobId, ControlledAiNotebookInstanceResource.class, user.getAuthenticatedRequest())
-        .getResult();
+    JobService.JobResultOrException<ControlledAiNotebookInstanceResource> creationResult =
+        jobService.retrieveJobResult(
+            createJobId,
+            ControlledAiNotebookInstanceResource.class,
+            user.getAuthenticatedRequest());
+    assertNull(creationResult.getException(), "Error creating controlled AI notebook instance");
+    assertNotNull(
+        creationResult.getResult(), "Unexpected null created controlled AI notebook instance");
```

Comment:
Maybe add in your notes from the PR description about known causes for this.

Reply:
I think that's a comment that'll get stale and not be especially helpful. If someone's looking at this test, it's because it failed and they have the cause in front of them.